stdlib/public/core/UnsafeBufferPointer.swift.gyb

```diff
@@ -328,8 +328,29 @@ public struct Unsafe${Mutable}BufferPointer<Element>
     self.init(start: slice.base.baseAddress! + slice.startIndex,
       count: slice.count)
   }
+
+  @_inlineable
+  public
+  init(_ other:UnsafeMutableBufferPointer<Element>)
+  {
+    self._position = UnsafePointer<Element>(other._position)
```

Comment:
Here and below, please follow prevailing style above; besides the style notes above (`public` on same line as `init`, space after `:`, `{` on same line as arguments), set internal members without prefixing `self` and do not align `_end =` with spaces.